drivers/SmartThings/zigbee-power-meter/src/ezex/init.lua

```diff
@@ -39,6 +48,11 @@ end
 local device_init = function(self, device)
   device:set_field(constants.SIMPLE_METERING_DIVISOR_KEY, 1000000, {persist = true})
   device:set_field(constants.ELECTRICAL_MEASUREMENT_DIVISOR_KEY, 10, {persist = true})
+
+  device:remove_monitored_attribute(SimpleMetering.ID, SimpleMetering.attributes.InstantaneousDemand.ID)
+  device:remove_configured_attribute(SimpleMetering.ID, SimpleMetering.attributes.InstantaneousDemand.ID)
+  device:add_monitored_attribute(instantaneous_demand_configuration)
+  device:add_configured_attribute(instantaneous_demand_configuration)
```

Comment:
I would have expected that you only need to call `add` and that the new config values would overwrite the default ones. Is that not the case?

Reply:
@greens I removed the line calling remove_XXX api!